src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -196,3 +200,66 @@
               :on-press            (fn []
                                      (rf/dispatch [:hide-bottom-sheet])
                                      (when on-press (on-press)))}]))]])))
+
+(defn get-reaction-authors-accessibility-id
+  [reaction-type]
+  (-> reaction-type
+      constants/reactions
+      name
+      (str "-reaction-count")))
+
+(defn contact-list-item-fn
+  [{:keys [from compressed-key]}]
+  (let [[primary-name secondary-name] (rf/sub [:contacts/contact-two-names-by-identity
+                                               from])
+        {:keys [ens-verified added?]} (rf/sub [:contacts/contact-by-address from])]
+    ^{:key (str compressed-key)}
+    [contact-list-item/contact-list-item
+     {:on-press #(rf/dispatch [:chat.ui/show-profile from])}
+     {:primary-name   primary-name
+      :secondary-name secondary-name
+      :public-key     from
+      :compressed-key compressed-key
+      :ens-verified   ens-verified
+      :added?         added?}]))
+
+(defn get-tabs-data
+  [reaction-authors selected-tab]
+  (mapv (fn [[reaction-type author-details]]
+          ^{:key (str reaction-type)}
```

Comment:
I'm not sure we need a unique key here, because this is not building hiccup.

Reply:
Done